drivers/counter/Kconfig.nxp_pcf85263a

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2022 Fraunhofer AICOS
+#
+
+config COUNTER_NXP_PCF85263A
+	bool "NXP PCF85263A RTC/Stop-watch"
+	depends on I2C
+	help
+	  Enable counter driver based on NXP PCF85263A I2C device.
+
+choice
+	prompt "PCF85263A INTA pin mode"
+
+config NXP_PCF85263A_INTA_CLK_OUT
```

Comment:
I think we may want to expose this setting via devicetree, since multiple PCF85263A devices could be present on a board (and the user might want the INTA pin to operate in different modes)